src/Core/Test/ModulePathTests.cs

```diff
@@ -104,42 +104,5 @@ public class ModulePathTests {
                 Assert.AreEqual(test.Expected, new ModulePath("abc", test.SourceFile, null).IsDebug, test.SourceFile);
             }
         }
-
-        /// <summary>
-        /// Verify that the analyzer has the proper algorithm for turning a filename into a package name
-        /// </summary>
-        [TestMethod, Priority(0)]
-        public void ModulePathFromFullPath() {
```

Comment:
test no longer needed?

Reply:
The code that this test was using isn't used anymore (in fact, it became irrelevant when `PathResolver` was first introduced, but I had no time to clean it).